logger/common.go

```diff
@@ -349,6 +383,7 @@ func (l *Logger) Read(
 						return err
 					}
 
+					atomic.AddUint64(&bytesSentToDst, uint64(len(curLine)))
```

Comment:
Should we just use int instead of uint? 

We seem to rely on len() for calculations which always returns int. Thus needing to constantly convert to uint64 because we are using the atomic Uint operation.

Reply:
`bytesSentToDst` is used to count the total number of bytes in 1 minute. The max of uint32 is 4294967295 so if we use uint32, the limit of bytes we can count in 1 min is 4GiB or so. I'm not 100% sure if that is large enough so to be safe I prefer uint64 which has larger range.

Reply:
Synced up offline. No concerns.